IslandLanding/IslandLanding/Converters/ImageConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Xamarin.Forms;
+
+namespace IslandLanding.Converters
+{
+ public class ImageConverter : IValueConverter
+  {
+
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+      string ButtonImage = "volume_off_24px.png";
+
+      if (value is string)
+      {
+        var imageValue = (string)value;
+        if (!string.IsNullOrEmpty(imageValue))
```

Comment:
You can shrink this to one line:
'if (value is string imageValue && !string.IsNullOrEmpty(imageValue))'

Reply:
okay